x-pack/legacy/plugins/apm/public/hooks/useAvgDurationByBrowser.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import theme from '@elastic/eui/dist/eui_theme_light.json';
+import { useFetcher } from './useFetcher';
+import { useUrlParams } from './useUrlParams';
+import { AvgDurationByBrowserAPIResponse } from '../../server/lib/transactions/avg_duration_by_browser';
+import { TimeSeries } from '../../typings/timeseries';
+
+const colors = [
+  theme.euiColorVis0,
+  theme.euiColorVis1,
+  theme.euiColorVis2,
+  theme.euiColorVis3,
+  theme.euiColorVis4,
+  theme.euiColorVis5,
+  theme.euiColorVis6,
+  theme.euiColorVis7,
+  theme.euiColorVis8,
+  theme.euiColorVis9
+];
```

Comment:
These are defined several times already. Perhaps it would make sense to add them as a constant in https://github.com/elastic/kibana/tree/c40b32010462aa6731d5c8a9312444228b3a819e/x-pack/legacy/plugins/apm/common

Examples:
https://github.com/elastic/kibana/blob/c40b32010462aa6731d5c8a9312444228b3a819e/x-pack/legacy/plugins/apm/server/lib/transactions/breakdown/constants.ts#L10-L22

https://github.com/elastic/kibana/blob/c40b32010462aa6731d5c8a9312444228b3a819e/x-pack/legacy/plugins/apm/server/lib/metrics/by_agent/java/gc/fetchAndTransformGcMetrics.ts#L28-L36

https://github.com/elastic/kibana/blob/c40b32010462aa6731d5c8a9312444228b3a819e/x-pack/legacy/plugins/apm/server/lib/metrics/transform_metrics_chart.ts#L16-L24

Btw. I like the modulus approach (`colors[index % colors.length]`) you took that ensures we'll loop through them regardless how many lines there are. In contrast to 
https://github.com/elastic/kibana/blob/c40b32010462aa6731d5c8a9312444228b3a819e/x-pack/legacy/plugins/apm/server/lib/metrics/by_agent/java/gc/fetchAndTransformGcMetrics.ts#L151

Reply:
Added this in e865ebe1b5.